kernel/configs/fragments/common/network.conf

```diff
@@ -0,0 +1,77 @@
+# Our networking requirements
+### FIXME - this probably needs a good review ###
```

Comment:
Nit: Missing issue url.

Reply:
I stuck all the notes, queries and fixme's in one Issue, and referenced that Issue URL throughout the config files. Saved opening a dozen closely related issues.